infrastructure/resource_group_level/modules/function-acbs.bicep

```diff
@@ -0,0 +1,172 @@
+param location string
+param environment string
+param containerRegistryName string
+param appServicePlanEgressSubnetId string
+param appServicePlanId string
+param privateEndpointsSubnetId string
+param storageAccountName string
+
+
+// These values are taken from GitHub secrets
+@secure()
+param secureSettings object = {
+  MULESOFT_API_KEY: 'test-value'
+  MULESOFT_API_SECRET: 'test-value'
+  MULESOFT_API_UKEF_TF_EA_URL: 'test-value'
+  APIM_MDM_KEY: 'test-value'
+  APIM_MDM_URL: 'test-value'
+  APIM_MDM_VALUE: 'test-value' // different in staging and dev
+}
+
+// These values are taken from an export of Configuration on Dev
+// TODO:FN-419 Add to GitHub secrets?
```

Comment:
I see that in https://github.com/UK-Export-Finance/dtfs2/pull/1767 these values will change. @abhi-markan 

I can either wait for your PR to be merged, merge that into my long-lived, branch, rebase this, and make the change now, or make the change in a future card / PR, with a TODO here. I'll do the latter by default!